docs(concepts): remove the instruction that consumes bundle-loader (#3809)
webpack/webpack.js.org: src/content/concepts/module-federation.md

+ ReactDOM.render(<App />, document.getElementById('root'));
```

> The below mentioned method works fine, but can have some limitations or drawbacks.

Reviewer: ```suggestion
This method works but can have limitations or drawbacks.
```
